docs/pages/desktop-access/getting-started.mdx

```diff
@@ -135,45 +202,166 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
 Right click `Require user authentication for remote connections by using
 Network Level Authentication`, edit, select **"Disable"** and "OK".
 
-### Enable the Smart Card service
 
-Teleport performs certificate based authentication by emulating a smart card.
-To enable the smart card service, select:
+## Step 3/6. Configure Group to allow Teleport connections
+
+Next, complete the service account's configuration by giving its corresponding Group the minimum necessary permissions for Teleport to work.
+First, open a PowerShell terminal and figure out the `$DomainDN` and `$TeleportLDAPGroupName` variables you'll need to use:
+
+`$DomainDN` is the 
+[distinguished name](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ldap/distinguished-names) representation of your Domain Controller's
+domain name. For example, for the domain name `domain-controller.example.com`, the distinguished name representation is `DC=domain-controller,DC=example,DC=com`.
+If you aren't sure, you can get the DN by running `(Get-ADDomain).DistinguishedName` in PowerShell.
+
+`$TeleportLDAPGroupName` is the Group name you defined in step 1.
+
+Run the PowerShell script below, replacing variable values where appropriate.
+
+```powershell
+$DomainDN="DC=domain-controller,DC=example,DC=com" # replace this with your domain
```

Comment:
In this case, the cluster name needs to be same as the FQDN of the Teleport proxy?

Reply:
No, that should say "Domain Controller domain"